doc/source/coding-style/index.rst

```diff
@@ -25,7 +25,7 @@ formatting with the libraries for the "big three" data science packages: `NumPy`
 The purpose of this section is not to repeat coding style documentation
 but rather to describe coding best practices applicable to the `PyAnsys`_ project when there are any
 delineations, clarifications, or additional procedures above and
-beyond `PEP 8`_. For example, this section provides a topic on deprecation
+beyond PEP 8. For example, this section provides a topic on deprecation
```

Comment:
I removed multiple links to the same topic. On any given page, you only want to link on the first use (unless the page is long and having another link much lower on the page would save the user from a lot of scrolling). Linking each time ends up giving the page a polka dot appearance.